util/new_keyboard.sh

```diff
@@ -13,165 +13,9 @@ echo_bold() {
     echo -e "\033[1m$1\033[m"
 }
 
-# Prompt the user for information, showing the default value in brackets.
-prompt() {
-    local message="$1"
-    local default="$2"
+# check for presence of Python 3.5 or greater
+# PYTHON_VERSION = $(python --version | )
```

Comment:
```suggestion

```

Since it's already commented out, I don't see much reason to leave it.

Reply:
I was thinking the old script could check if the user has the requisite toolchain to run this script before executing, and maybe even install it first. I've no idea how to do that though, or if it's even worth our time.

Reply:
The global cli already does it, so I see little benefit in doing it, too:
https://github.com/qmk/qmk_cli/blob/master/qmk_cli/script_qmk.py#L69-L71

Reply:
Are there any checks that trigger in situations where the user doesn't have *any* version of Python installed?

Reply:
I'm not aware of any.